conf/drillTestConfig.properties.example

```diff
@@ -86,6 +86,37 @@
 ########################################
 #RESTART_DRILL_SCRIPT=framework/scripts/restart_drill_script
 
+########################################
+# Authentication Mechanisms
+########################################
+#AUTH_MECHANISM=MAPRSASL
+#AUTH_MECHANISM=PLAIN
+
+########################################
+# Is HTTPS enabled
+########################################
+#HTTPS_ENABLED=true
+
+########################################
+# Is SSL enabled
+########################################
+#SSL_ENABLED=true
+
+########################################
+# Truststore
+########################################
+#TRUSTSTORE_PATH=/opt/mapr/conf/ssl_truststore
+
+########################################
+# Truststore Password
+########################################
+TRUSTSTORE_PASSWORD=mapr123
+
+########################################
+# Kerberos Principal
+########################################
+KERBEROS_PRINCIPAL=
```

Comment:
Do we expect a default value or an example here?

Reply:
done